sdk/loadtestservice/Azure.Developer.LoadTesting/tests/Samples/Sample1_CreateOrUpdateTest.cs

```diff
@@ -47,20 +37,24 @@ public void CreateOrUpdateTest()
                     engineInstances = 1,
                     splitAllCSVs = false,
                 },
-                secrets = new {
+                secrets = new
+                {
                     secret1 = new
                     {
                         value = "https://sdk-testing-keyvault.vault.azure.net/secrets/sdk-secret",
```

Comment:
Does this have to be a real secret path, or can it be anything? If so, we should factor this out into a variable so that it's more obvious that it must be replaced with a real value.

Reply:
@christothes this is URI to some KeyVault holding the secret, that does make sense, updating this in the latest commit.

Reply:
Yes, I was just wondering if the sample would fail if this was not a legitimate secret URI. If not, then it is fine as is.